environment.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, image, true_bboxes, use_gpu=False):
         self.true_bboxes = true_bboxes
         self.history = self.create_empty_history()
         self.bbox = np.array([0, 0, image.width, image.height])
-        self.iou = 0
+        self.iou = self.compute_best_iou()
```

Comment:
Initializing iou with zero is wrong, the initial bounding box covers the entire image, so it also covers the target box, thus the initial iou is greater than 0. I changed it, because it lead to the first reward always being 0 or 1, even if -1 would have been the correct reward.